src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderService.java

```diff
@@ -97,35 +97,42 @@ public ProbationRecord getProbationRecord(String crn, boolean applyDocumentFilte
             .collectSortedList(new ConvictionBySentenceComparator()
                                 .thenComparing(Conviction::getConvictionId));
 
-        // This Mono resolves to a 4 tuple containing the convictions (see above), offender managers and CRN documents
+        // This Mono resolves to a 5 tuple containing the convictions (see above), offender managers, CRN documents and assessments
         // As a check against exclusions only we call getOffender which will return 403 status
-        Mono<Tuple4<List<Conviction>, List<OffenderManager>, GroupedDocuments, CommunityApiOffenderResponse>> communityApiMono = Mono.zip(
+        var communityApiMono = Mono.zip(
```

Comment:
Not really just a community API mono but I can't think of anything better :-)   If we are using `var` here then rename the Mono<List<Conviction>> above

Reply:
Good spot on these, I had meant to tidy these up but then never got round to it. I'll push a few changes around general tidying and readability